models/klaviyo__persons.sql

```diff
@@ -0,0 +1,25 @@
+with person as (
+
+    select *
+    from {{ var('person') }}
+),
+
+person_metrics as (
+
+    select *
+    from {{ ref('int_klaviyo__person_metrics') }}
+),
+
+person_join as (
+
+    select
+        person.*,
+        {{ dbt_utils.star(from=ref('int_klaviyo__person_metrics'), except=['person_id'] if target.type != 'snowflake' else ['PERSON_ID'] ) }}
```

Comment:
Would `lower(person_id)` worker here instead of the conditional?

Reply:
yeah so it's not about the casing of the value of the `person_id` (which is what the `lower()` function would change), it's about the casing of the column name in the database (`person_id` vs `PERSON_ID`).